calculator/unit_data.txt

```diff
@@ -86,28 +91,29 @@
 # frequency
   Hz: Hertz / Hertz
   
+  
 # power
-  W: Watt / Watt
+  W: Watt
  
  
 # electric charge
-  C: Coulomb / Coulomb
+  C: Coulomb
  
  
 # electric potential
-  V: Volt / Volt
+  V: Volt
   
  
 # electric capacitance
-  F: Farad / Farad
+  F: Farad
   
   
 # resistance
-  Ohm: Ohm / Ohm
+  Ohm: Ohm / Ohme
```

Comment:
As far as I can tell the English plural form of Ohm is Ohms (https://en.wiktionary.org/wiki/ohm#:~:text=Noun,ohm%20(plural%20ohms))